### PR TITLE
Add description of Fivox applications

### PR DESCRIPTION
**Pull request by jafyvilla**

### CODE REVIEW

**Comment by jafyvilla** [comment]
@eile I wasn't sure where to put this. Have a look and let me know if more detail is needed for each of the cases described.

**Comment by eile** [comment]
Can we have an example image for each of the three main use cases?

File: doc/userGuide/applications.md [hunk 1]
```diff
@@ -0,0 +1,61 @@
...
+#### Local Field Potential
+
+Compute the LFP signal at the extracellular space of a given circuit, based on
+a simulation current report
+([more info](https://en.wikipedia.org/wiki/Local_field_potential)).
+
+The output is generated as a 3D volume, containing each of the voxels the LFP
+value corresponding to the position at the center of the voxel.
+
+For convenience, a separate tool (sample-point) is provided, to generate the
```
Comment (eile) [inline doc/userGuide/applications.md]: It's more than convenience.

File: doc/userGuide/applications.md [hunk 2]
```diff
@@ -0,0 +1,61 @@
...
+time series of the output voltage at a specific 3D point. In this case the space
+is not voxelized, and the computation is done for a single point, so the
+resulting values are more accurate, as an exact 3D position is used as input.
```
Comment (eile) [inline doc/userGuide/applications.md]: Are they more precise if the point is the same as a voxel center? If so, why?

**Comment by jafyvilla** [comment]
Hold the merge, I'm trying to find a way to make images visible both in github and doxygen

**Pull request merged**